Fix layout issues around images on the About page for narrow screens (#471)

marceloams opened the pull request:
Closes #471. In narrow screens (mobile), the images are now displayed above the text for improved readability.

wbazant commented:
Thanks for making the changes, the text is more readable now! Would you be up for trying flex layouts?


![Screenshot from 2024-10-05 08-51-42](https://github.com/user-attachments/assets/dffea1bd-e75a-4c8f-9e98-0e7e17e1bd83)
The pictures don't need to be up to 160px if they're not to the side of the text, this 2x2 grid could take the whole width. 

I think this would look even better in a flex layout too:  the staff member's picture to the right, their name and email to the left, centered vertically, and jointly taking the whole width.
![Screenshot from 2024-10-05 08-52-15](https://github.com/user-attachments/assets/d0c6eda8-14a2-4322-b51c-e0a360603d87)

marceloams commented:
Hey @wbazant,

I've implemented the changes you requested for the grid. I also set a max height to prevent it from becoming too large, as you can see below:

![Captura de tela de 2024-10-06 13-32-11](https://github.com/user-attachments/assets/17228263-0d08-46a1-9174-065577ab7727)

For the staff members section, this is the current result:

![Captura de tela de 2024-10-06 13-05-09](https://github.com/user-attachments/assets/d0b28c5a-e214-4b56-b0d3-aebf5a46551b)

However, I'd like to propose an alternative approach, where the image remains on the left and the text on the right. When the text exceeds the height of the image, it wraps underneath, taking up the full width, as shown below:

![Captura de tela de 2024-10-06 13-26-26](https://github.com/user-attachments/assets/512451c0-6cc6-4680-a2e3-0e808a89a99e)

I added a justify alignment for the text in this layout, but the left alignment works well too. Let me know your preference!

wbazant commented:
@marceloams thanks for making the grid take up the full width, it does look better on the screenshot. For the second part, I think your solution works better than what I proposed - once the director's name /title/ email fits on the right, then the text underneath will fit too. Would you push those changes to the branch?

marceloams commented:
@wbazant I've completed the requested changes. Please take a look when you have a moment, and I look forward to your feedback!

wbazant commented:
Thanks, that renders itself well on various screen sizes! I have an amendment to suggest but opening a PR to your fork felt like overkill here, so I merged your commit and opened https://github.com/falling-fruit/falling-fruit-web/pull/491 .

Would you be up for doing some other work on the project? The next thing to do for the About page is probably #492 , or you're welcome to grab any other issue.
